app/rts/build.sh

```diff
@@ -4,12 +4,22 @@ set -o errexit
 
 cd "$(dirname "$0")"
 rm -rf dist/
-yarn install --frozen-lockfile
+# Install node modules if not already installed
+# This is required for the first time build as node_modules is not present in the image
+! test -d "/node_modules" && yarn install --frozen-lockfile
```

Comment:
This test will always be `false`, since it's looking for a `node_modules` folder in the root of the filesystem. This is equivalent to just `yarn install --frozen-lockfile`, which is what we want here, right? Irrespective of whether a `node_modules` already exists or not? 🤔

```suggestion
yarn install --frozen-lockfile
```

Reply:
Right, i have updated.